README.md

```diff
@@ -189,7 +187,7 @@ The public key of the remote peer.
 
 #### `socket.publicKey`
 
-The connections public key.
+The public key of the local socket.
```

Comment:
What about this? Just above there is "remotePublicKey", so at the end it looks like this:

#### `socket.remotePublicKey`
The public key of the remote peer.

#### `socket.publicKey`
The public key of the local socket.